docker/buildkite/docker-compose-es7.yml

```diff
@@ -0,0 +1,69 @@
+version: "3.5"
+
+services:
+  cassandra:
+    image: cassandra:3.11
+    networks:
+      services-network:
+        aliases:
+          - cassandra
+
+  zookeeper:
+    image: wurstmeister/zookeeper:3.4.6
+    networks:
+      services-network:
+        aliases:
+          - zookeeper
+
+  kafka:
+    image: wurstmeister/kafka:2.12-2.1.1
+    depends_on:
+      - zookeeper
+    networks:
+      services-network:
+        aliases:
+          - kafka
+    environment:
+      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://kafka:9092
+      KAFKA_LISTENERS: PLAINTEXT://0.0.0.0:9092
+      KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
+
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch:7.9.3
```

Comment:
Shall we use the oss version (`elasticsearch-oss`)  to avoid any potential license issues?

Reply:
yeah good catch